src/open_inwoner/utils/forms.py

```diff
@@ -163,3 +164,56 @@ def clean(self, *args, **kwargs):
                 )
 
         return f
+
+
+class MathCaptchaField(forms.Field):
+    def __init__(
+        self,
+        range_: tuple = (1, 10),
+        operators: list[str] | None = None,
+        *args,
+        **kwargs,
+    ):
+        super().__init__(*args, **kwargs)
+        self.widget = forms.TextInput()
+        self.range_ = range_
+        self.operators = operators or ["+", "-"]
+        self.question, self.answer = self.generate_question_answer_pair(
+            self.range_, self.operators
+        )
+
+    @staticmethod
+    def generate_question_answer_pair(
+        range_: tuple[int, int],
+        operators: list[str],
+    ) -> tuple[str, int]:
+        lower, upper = range_
+        num1 = random.randint(lower, upper)  # nosec
+        num2 = random.randint(lower, upper)  # nosec
+        operator = random.choice(operators)  # nosec
```

Comment:
`secrets.choice(range(m, n))` works as well. However, is there a point outside of cryptographic or other security-sensitive contexts?

Reply:
I ended up following your suggestion after all. Since performance is not an issue, it's better to default to the secure option, also for future cases.